tools/generate-contributors.ts

```diff
@@ -78,40 +133,16 @@ async function main(): Promise<void> {
 
   // fetch the user info
   const users = await Promise.all(
-    githubContributors.map(async c => {
-      const response = await fetch(c.url, { method: 'GET' });
-      return (await response.json()) as User;
-    }),
+    githubContributors
+      // remove ignored users and bots
+      .filter(
+        usr => usr.login && usr.type !== 'Bot' && !IGNORED_USERS.has(usr.login),
+      )
```

Comment:
we actually don't have to drop this contributors cli, all what has to be done is this change as some users may not have their profiles set to public, 

with latest github api we are calling those endpoints only to retrieve "name", as all other data is available already to us from contributions api

Reply:
more than happy to remove a dependency here!